### PR TITLE
Issue #132: Fix menu link to properly display Login/logout link as ECL.

### LINKED ISSUE #132: Fix menu link
Add missing ECL classes.

### PR DESCRIPTION
**Pull request by aritomelo**
Fix #132

### CODE REVIEW

File: templates/menu_link/menu_link.component.inc [hunk 1]
```diff
@@ -152,10 +152,11 @@ function ec_europa_preprocess_menu_link__menu_nexteuropa_site_links__footer(arra
  */
 function _ec_europa_menu_link_footer_alter(array $variables) {
   $variables['atomium']['attributes']['wrapper']->append('class', 'ecl-footer__menu-item');
-  $variables['output']['#options']['attributes']['class'][] = 'ecl-footer__link';
+  $variables[$variables['base_hook']][$variables['base_hook']]['#options']['attributes']['class'][] = 'ecl-footer__link';
+  $variables[$variables['base_hook']][$variables['base_hook']]['#options']['attributes']['class'][] = 'ecl-link--inverted';
 
-  $classes = &$variables['output']['#options']['attributes']['class'];
-  if (!empty($classes) && \in_array('ecl-icon', $classes, TRUE)) {
```
Comment (drupol) [inline templates/menu_link/menu_link.component.inc]: Why this has been removed ?

**Comment by drupol** [comment]
Do we need a backport in branch 0.x ?

File: templates/menu_link/menu_link.component.inc (hunk 1 above)
Comment (aritomelo) [reply templates/menu_link/menu_link.component.inc]: Have to check!

**Comment by drupol** [comment]
There is no need to backport this into 0.x, it's already there.

**Pull request merged**